components/waitwhile/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.2",
```

Comment:
I'm having issues with my formatString and formatArrayString functions in the common/utils.js.  I get an 'Object malformed' error when I test the create-customer action.  Can I remove these functions because they are not working.  Can someone in QA check?

Reply:
I don't see a problem in removing those functions. The API should return an HTTP error if something malformed is sent, shouldn't it?